src/test/java/org/opensearch/performanceanalyzer/net/NetClientTest.java

```diff
@@ -34,6 +35,7 @@
 @PowerMockIgnore({"org.apache.logging.log4j.*", "com.sun.org.apache.xerces.*"})
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({NetClient.class, InterNodeRpcServiceGrpc.InterNodeRpcServiceStub.class})
+@Ignore("See please https://github.com/opensearch-project/performance-analyzer-rca/issues/529")
```

Comment:
@ansjcy we need to work on PowerMock removal, https://github.com/opensearch-project/performance-analyzer-rca/issues/529, it is not playing well with recent JDKs (and is dead essentially)